packages/url_launcher/url_launcher/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 * Fix test button check for iOS 15.
 
+## 6.0.8
+
+* Adding API level 30 required package visibility configuration to the example's AndroidManifest.xml and README
+
```

Comment:
Now that this is going to cause the package to be published as 6.0.8, the `## NEXT` changelog entry can be added to `## 6.0.8`, resulting in:

```
## 6.0.8

* Adding API level 30 required package visibility configuration to the example's AndroidManifest.xml and README
* Fix test button check for iOS 15.
```

(And no `## NEXT` entry for now!)

Reply:
(Filed https://github.com/flutter/flutter/issues/85584 to have CI catch this)

Reply:
Good catch, I pushed a merged version.